src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,106 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number,
+  perPage: number,
+  currentPage: number,
+  onPageChange: (page:number) => void,
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const getNumberOfPages = Math.ceil(total / perPage);
+  const arrayOfPages = getNumbers(1, getNumberOfPages);
+
+  const activePage = (
```

Comment:
Why do you need this function?
You could use `currentPage` instead of calculating `activePage`. 
I don't think if you can have currentPage bigger than the total pages.